src/metatrain/cli/export.py

```diff
@@ -112,17 +128,37 @@ def export_model(
 
     :param model: model to be exported
     :param output: path to save the model
+    :param extensions: path to save the extensions
     :param metadata: metadata to be appended to the model
     """
     path = str(
         Path(check_file_extension(filename=output, extension=".pt"))
         .absolute()
         .resolve()
     )
-    extensions_path = str(Path("extensions/").absolute().resolve())
+
+    if _has_extensions():
+        extensions_path = str(Path(extensions).absolute().resolve())
+    else:
+        extensions_path = None
 
     if not is_atomistic_model(model):
         model = model.export(metadata)
 
     model.save(path, collect_extensions=extensions_path)
-    logging.info(f"Model exported to '{path}' and extensions to '{extensions_path}'")
+    if extensions_path is not None:
+        logging.info(
+            f"Model exported to '{path}' and extensions to '{extensions_path}'"
+        )
+    else:
+        logging.info(f"Model exported to '{path}'")
+
+
+def _has_extensions():
+    """Check if any torch extensions are currently loaded (besides metatensor)."""
+    loaded_libraries = torch.ops.loaded_libraries
```

Comment:
Isn't this something that we should to upstream in metatensor?

Reply:
@Luthaf

Reply:
Ah, ok =) I personally prefer the code always doing the same thing, so wrapper scripts don't have to worry about specific models having extensions or not, they can always set this and always carry the directory around. Having an extra empty directory in some cases is fine for me

Reply:
Okay I am fine with either way, but I think we should be consistent, right?

Reply:
I can see the point for not doing it in metatrain, since we expect that users will directly interact with the code and models manually

Reply:
Yes. Maybe lets keep the behavior in `metatrain` as is and do a discussion upstream?

Reply:
Sounds good